tests/smoke/pdm.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+SMOKE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+. ${SMOKE_DIR}/base.sh build_system=pdm
```

Comment:
we change the call for this base.sh .. could you please rebase your code and update this line. 

```suggestion
. ${SMOKE_DIR}/base.sh build_system pdm
```